generators/client/templates/angular/src/main/webapp/app/admin/metrics/metrics-cache/metrics-cache.component.html.ejs

```diff
@@ -19,18 +19,18 @@
 <h3 <%= jhiPrefix %>Translate="metrics.cache.title">Cache statistics</h3>
 
 <div class="table-responsive" *ngIf="!updating && cacheMetrics">
-    <table class="table table-striped">
+    <table class="table table-striped" aria-describedby="cacheMetrics">
```

Comment:
In https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-describedby_attribute is written that `aria-describedby` value is:
```
a space-separated list of element IDs
```
But there is no element with ID `cacheMetrics`. Probably need to add `id="cacheMetrics"` to `h3` element in this component to make it work correctly.
Same comment for all other `aria-describedby` changes.